Library/Formula/srm.rb

```diff
@@ -0,0 +1,76 @@
+require "formula"
+
+class Srm < Formula
+  homepage "http://www.thc.org/"
+  url "https://www.thc.org/releases/secure_delete-3.1.tar.gz"
+  sha1 "ce6391003587b37bd3713a967de04de1f0759107"
+
+  keg_only :provided_by_osx if MacOS.version >= :mavericks
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!